lib/pipeline/shared.go

```diff
@@ -111,15 +111,11 @@ func readerContainerAndVolume(rr *unstructured.Unstructured) (v1.Container, v1.V
 		Image:   "bitnami/kubectl:1.20.10",
 		Command: []string{"sh", "-c", resourceRequestCommand},
 		VolumeMounts: []v1.VolumeMount{
-			{
-				MountPath: "/output",
-				Name:      "vol0",
-			},
+			{MountPath: "/output", Name: volumeName},
```

Comment:
should we update ours to use the same pattern just for consistency? Tiny thing but I could imagine future us getting mixed up